README.md

```diff
@@ -64,7 +64,24 @@ To get the source code from GitHub, type
 1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
```

Comment:
Can you move this to the "Add requirement and install with one command" section?